src/main/resources/locales/cy/error.json

```diff
@@ -1,4 +1,6 @@
 {
-    "header": "Mae rhywbeth wedi mynd o'i le",
+    "header": "Sorry, there is a problem with the service",
+    "retryMessage": "Please try again later.",
```

Comment:
These two lines end with full stops, however the ticket screenshot does not. Could you double check which one is needed?

Reply:
Chris C has confirmed we need the full stops. They are sentences within the body not heading.